static/js/src/modules/feedback.js

```diff
@@ -0,0 +1,66 @@
+const { el, mount, textContent, list, setChildren, setStyle, setAttr } = redom
+
+let pageTitle = document.querySelector(".blog-post-title")
+
+let title = `Potential Missing Documentation: ${pageTitle.textContent}`
+let body = `
+Describe the missing content:
+
+Where would you have expected to find the information:
+
+`
+
+const newIssueQuery = `title=${encodeURIComponent(title)}&body=${encodeURIComponent(body)}`
+
+class FeedBack {
```

Comment:
Can we add an ignore or close button to make this box disappear?
There may be situations where a user does not want this sort of help (or has already filed one or more issues).